spec/models/queen_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Queen, type: :model do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:queen) { FactoryGirl.create(:queen, :is_on_board) }
+
+  describe 'queen#valid_move?' do
```

Comment:
The "best practice" for describe blocks would be

`describe '#valid_move?' do`